internal/middleware/logging.go

```diff
@@ -2,65 +2,103 @@ package middleware
 
 import (
 	"context"
-	"time"
+	"encoding/json"
 
 	"github.com/openfga/openfga/pkg/logger"
 	serverErrors "github.com/openfga/openfga/pkg/server/errors"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 )
 
+const (
```

Comment:
I like this approach with the `grpc_` prefix, because I can see the community asking us to add logged fields for the HTTP variations, which we'd have to implement in the grpc-gateway `runtime` middleware for the grpc-gateway HTTP --> gRPC transcoding process. In that case we'd also introduce `http_method`, `http_path`, etc..

So we'd have example values such as

`grpc_service: "openfga.v1.OpenFGAService"`
`grpc_method: "Check"`
`grpc_type: "unary"
`grpc_code: "OK"`, `grpc_code: "Code(2002)"`, etc..

`http_method: "POST"`
`http_path: "/stores/:id/check"`
`http_status: 200` etc..